src/Hl7.Fhir.R5.Core/Specification/Source/FhirPackageSource.cs

```diff
@@ -97,14 +97,14 @@ public IEnumerable<string> ListResourceUris(ResourceType? filter = null)
         }
 
         ///<inheritdoc/>
-        public async Task<IEnumerable<ConceptMap>?> FindConceptMapsAsync(string? sourceUri = null, string? targetUri = null)
+        public async Task<IEnumerable<ConceptMap>> FindConceptMapsAsync(string? sourceUri = null, string? targetUri = null)
         {
             var resources = await _resolver.FindConceptMaps(sourceUri, targetUri).ConfigureAwait(false);
-            return resources == null ? null : resources.Cast<ConceptMap>();
+            return resources.Cast<ConceptMap>();
         }
 
         ///<inheritdoc/>
-        public IEnumerable<ConceptMap>? FindConceptMaps(string? sourceUri = null, string? targetUri = null)
+        public IEnumerable<ConceptMap> FindConceptMaps(string? sourceUri = null, string? targetUri = null)
```

Comment:
I think these changes are breaking changes (return type). So we have to document that.

Reply:
Done.